clients/drcachesim/tracer/physaddr.h

```diff
@@ -56,7 +56,7 @@ class physaddr_t {
     // 0 is a possible valid physical address, as are large values beyond
     // the amount of RAM due to holes in the physical address space.
     bool
-    virtual2physical(addr_t virt, OUT addr_t *phys);
+    virtual2physical(addr_t virt, OUT addr_t *phys, OUT bool *from_cache = nullptr);
```

Comment:
Document what exactly `from_cache` means -- that it depends on `-virt2phys_freq`.

Reply:
Done.